grammar.js

```diff
@@ -92,9 +91,14 @@ module.exports = grammar({
 
     // A function, generator, class, or variable declaration
     _declaration: $ => prec(PREC.DECLARATION, choice(
-      $.function,
-      $.generator_function,
-      $.class,
+      seq(
```

Comment:
@maxbrunsfeld Do you remember what this was for? Since we override `_declaration` in the typescript grammar, I'm not sure if I should bring this in or not.